_partials/_devops-mcp-commands.md

```diff
@@ -0,0 +1,30 @@
+
+You can use the following commands with $MCP_LONG.
+
+| Command                         | Parameter         | Required | Description                                                                                                                                                                                                                                                                                                     |
```

Comment:
This whole table would benefit from variables for services, some rephrasing for readability and clarity, definite articles, and code font for examples and defaults.